src/tests/regress/data/expected/float8.out

```diff
@@ -616,7 +616,15 @@ ERROR:  cannot take logarithm of zero
 SELECT ln(f.f1) from FLOAT8_TBL f where f.f1 < '0.0' ;
 ERROR:  cannot take logarithm of a negative number
 SELECT exp(f.f1) from FLOAT8_TBL f;
-ERROR:  value out of range: underflow
+          exp          
+-----------------------
+                     1
+                     0
+                     0
+ 7.399123060905129e-16
```

Comment:
both float64, but RW works while PG underflows....?

Reply:
I'd prefer comment this test case out (`--@ ` in input `.sql` file) rather than updating its expected output, which may give a wrong signal of being compatible.

As for this issue, PG does have some overflow/underflow checking that we lack: #6826

BTW the `select exp(-10000000);` test case (and float variant `select exp(-10000000::float8);`) in sqllogictest also underflows in PG. But I think we can keep it there and acknowledge in comments it is not PG-compatible right now.

To summarize, `sqllgocitest` to document our current behavior and `regress` tests to keep record of our compatibility. Does this make sense?

Reply:
I see, it makes sense, thanks for the detailed explanation